README.md

```diff
@@ -4,59 +4,86 @@
 
 # Gitbase Playground
 
-Web application to query git repositories using SQL. Powered by [gitbase](https://github.com/src-d/gitbase).
+Web application to query Git repositories using SQL. Powered by [gitbase](https://github.com/src-d/gitbase), it allows to perform SQL queries about the Git history and the [Universal AST](https://doc.bblf.sh/) of the code itself.
 
 ![Screenshot](.github/screenshot.png?raw=true)
 
-
 # Usage
 
-## Dependencies
+## With Docker Compose
+
+The easiest way to run Gitbase Playground and its dependencies is using [Docker Compose](https://docs.docker.com/compose/install/).
 
-The playground will run the queries against a [gitbase](https://github.com/src-d/gitbase) server, and will request UASTs to a [bblfsh](https://doc.bblf.sh/) server; both should be accessible for the playground; you can check its default [configuration values](docs/CONTRIBUTING.md#configuration).
+The first step is to populate a directory with some Git repositories to be served by gitbase before running it. For example:
 
+```bash
+$ mkdir $HOME/repos
+$ cd $HOME/repos
+$ git clone git@github.com:src-d/gitbase.git
+$ git clone git@github.com:bblfsh/bblfshd.git
+$ git clone git@github.com:src-d/gitbase-playground.git
+```
 
-## Run the Playground
+Next you will need to download the `docker-compose.yml` file from this repository and run `docker-compose`. This tool will run three different containers: the playground frontend itself, gitbase, and bblfshd. To kill the running containers use `Ctrl+C`.
+
+```bash
+$ wget https://raw.githubusercontent.com/src-d/gitbase-playground/master/docker-compose.yml
+$ docker-compose pull
```

Comment:
The need to `wget` something in the README instructions looks quite shady :/

As README is a part of the repository, do you think we could assume it has been already cloned to the local FS? 
If that is absolutely necessary to verify this assumption - we could replaces `wget` with `ls -la docker-compose.yml` to help user diagnosis this problem early on.

Otherwise, I'm afraid that we might need to start a docs with something like:
 - buy a computer
 - connect it to internet
 - ... 

What do you think?

Reply:
JFYI: I never read readme from local FS. Only on github. And in 90% I don't clone the repo but use instruction how to install something using package manager / script / other tool.

Reply:
@smacker In this case, would not it be natural to assume that such user would be well-aware about ability to wget things from GH?

Up to you @carlosms, what you think is better here.

Reply:
Seriously, every time when I need to install homebrew I open https://brew.sh/ with curl command instead of looking for a file by myself :-D

For me must weirder looks:
```
To kill the running containers use `Ctrl+C`.
```

but as you wish.

Reply:
I don't see it any more shady than downloading a binary from the releases section, at least this is a text file you can read. I've seen other projects where the install instruction are `curl  https://raw.githubusercontent.com.... | bash`, and personally I find it much more convenient than having to clone the repo to execute a small file.

If you prefer we can replace this with something like:

> Download the [docker-compose.yml](https://github.com/src-d/gitbase-playground/blob/master/docker-compose.yml), `cd` to the directory where you put it, and execute...

But that just feels less convenient to me.

> As README is a part of the repository, do you think we could assume it has been already cloned to the local FS?

To me the README is more like the landing page for this project. I would not assume the reader cloned the repo unless they plan to run from sources. I'd prefer to keep the README for users, and CONTRIBUTING to run from sources.

Plus cloning the repo has the requirement that it should be placed inside your `$GOPATH` if you want to compile later. Something too involved for first-time users that may want to try it out quickly.